fighthealthinsurance/chat_interface.py

```
@@ -852,26 +852,21 @@ async def handle_chat_message(
-                    final_context_part = context_part
-                    break
-            except Exception as e:
-                await asyncio.sleep(0.1)
-                model_name = getattr(model_backend, "model_name", "Unknown Model")
-                logger.opt(exception=True).debug(
-                    f"Error with model {model_name} during chat generation: {e}"
-                )
+            if response_text and response_text.strip():
+                final_response_text = response_text.strip()
+                final_context_part = context_part
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)